esti/hooks_test.go

```diff
@@ -461,6 +460,9 @@ func parseAndUploadActions(t *testing.T, ctx context.Context, repo, branch strin
 		require.NoError(t, err)
 		require.Equal(t, http.StatusCreated, resp.StatusCode())
 	}
+
+	// wait 10 seconds to let the actions cache expire.
+	time.Sleep(10 * time.Second)
```

Comment:
not doing good to the other work of reduce tests running time.

Reply:
I agree, but it's a system test where we have cache we don't want to override.